crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -54,6 +57,12 @@ def supported?
 
     protected
 
+    def network_json30
```

Comment:
I would prefer this method name to be a verb.  Currently the name doesn't give any indication of what it does, only that it's related to `network.json` from 3.0.